pkg/services/live/features/dashboard.go

```diff
@@ -16,12 +16,12 @@ import (
 type actionType string
 
 const (
-	ACTION_SAVED     actionType = "saved"
-	ACTION_DELETED   actionType = "deleted"
-	EDITING_STARTED  actionType = "editing-started"
-	EDITING_FINISHED actionType = "editing-finished"
+	ActionSaved    actionType = "saved"
```

Comment:
I assume this is go best practice :shrug:

Reply:
https://golang.org/doc/effective_go#mixed-caps - MixedCaps mostly recommended, Goland draws a warn, I think Protobuf generates consts with underscores though.